IFC4x3/PropertyConstants/t/TONGUE_GROOVE_3$cx1SJoTDuQBpEoCXtIYt.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TONGUE_GROOVE_3$cx1SJoTDuQBpEoCXtIYt" Name="TONGUE_GROOVE" UniqueId="ff9bb05c-4f27-4de1-a2f3-3b2321dd28b7" />
```

Comment:
Description.

Reply:
Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)